<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctClass</class>
 <widget class="QMainWindow" name="ctClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ct</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QVTKOpenGLNativeWidget" name="qvtk_1"/>
    </item>
    <item row="0" column="1">
     <widget class="QVTKOpenGLNativeWidget" name="qvtk_2"/>
    </item>
    <item row="1" column="0">
     <widget class="QVTKOpenGLNativeWidget" name="qvtk_3"/>
    </item>
    <item row="1" column="1">
     <widget class="QVTKOpenGLNativeWidget" name="qvtk_4"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="action3D"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action3D">
   <property name="text">
    <string>3D显示</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>qvtkopenglnativewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ct.qrc"/>
 </resources>
 <connections/>
</ui>
